### PR TITLE
change do60 config

### PR DESCRIPTION
**Pull request by sennin32**

### CODE REVIEW

File: keyboards/do60/do60.h [hunk 1]
```diff
@@ -7,24 +7,25 @@
...
 inline void do60_caps_led_off(void)   { DDRB &= ~(1<<2); PORTB &= ~(1<<2); }
 inline void do60_bl_led_off(void)   	{ DDRF &= ~(1<<4); PORTF &= ~(1<<4); }
+*/
 
-
+inline void setdefaultrgb(void){ rgblight_sethsv(100,100,100); }
 
 /* Do60 Keymap Definition Macro */
-#define LAYOUT_all( \
+#define KEYMAP( \
```
Comment (drashna) [inline keyboards/do60/do60.h]: Could this be changed back to `LAYOUT_all`?

File: keyboards/do60/do60.h [hunk 2]
```diff
@@ -38,18 +39,4 @@ inline void do60_bl_led_off(void)   	{ DDRF &= ~(1<<4); PORTF &= ~(1<<4); }
     { K40,  K41,  K42,  KC_NO,KC_NO,K45,  K46,  K47,  KC_NO,KC_NO,K4A,  K4B,  K4C,  K4D,   K4E }  \
 }
```
Comment (drashna) [inline keyboards/do60/do60.h]: Could the `LAYOUT_chiwi60_default` block be re-added?

File: keyboards/do60/keymaps/default/keymap.c [hunk 3]
```diff
@@ -4,25 +4,25 @@
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
 
   // 0: Base Layer
-  [0] = LAYOUT_all(
+  [0] = KEYMAP(
```
Comment (drashna) [inline keyboards/do60/keymaps/default/keymap.c]: Could these be changed back to `LAYOUT_all`?

File: keyboards/do60/keymaps/default/keymap.c [hunk 4]
```diff
@@ -4,25 +4,25 @@
...
       KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,   KC_5,   KC_6,   KC_7,   KC_8,   KC_9,    KC_0,    KC_MINS,  KC_EQL,  KC_GRV,   KC_BSPC,   \
       KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,   KC_T,   KC_Y,   KC_U,   KC_I,   KC_O,    KC_P,    KC_LBRC,  KC_RBRC,           KC_BSLS,   \
       KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,   KC_G,   KC_H,   KC_J,   KC_K,   KC_L,    KC_SCLN, KC_QUOT,  KC_NO,             KC_ENT,    \
-      KC_LSFT, KC_NO,   KC_Z,    KC_X,    KC_C,   KC_V,   KC_B,   KC_N,   KC_M,   KC_COMM, KC_DOT,  KC_RSFT,  KC_NO,   KC_UP,    KC_SLSH,    \
+      KC_LSFT, KC_NO,   KC_Z,    KC_X,    KC_C,   KC_V,   KC_B,   KC_N,   KC_M,   KC_COMM, KC_DOT,  KC_RSFT,  KC_SLSH,   KC_UP,    KC_SLSH,    \
       KC_LCTL, KC_LGUI, KC_LALT,                  KC_SPC, KC_NO,  KC_BSPC,                 KC_RGUI, F(0),     KC_LEFT, KC_DOWN,  KC_RIGHT),
 
   // 1: Function Layer
-  [1] = LAYOUT_all(
+  [1] = KEYMAP(
```
Comment (drashna) [inline keyboards/do60/keymaps/default/keymap.c]: Could these be changed back to `LAYOUT_all`?

File: keyboards/do60/keymaps/test/keymap.c [hunk 5]
```diff
@@ -4,15 +4,15 @@
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
 
   // 0: Base Layer
-  [0] = LAYOUT_all(
+  [0] = KEYMAP(
```
Comment (drashna) [inline keyboards/do60/keymaps/test/keymap.c]: Could these be changed back to `LAYOUT_all`?

File: keyboards/do60/keymaps/test/keymap.c [hunk 6]
```diff
@@ -4,15 +4,15 @@
...
       KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,   KC_5,   KC_6,   KC_7,   KC_8,   KC_9,    KC_0,    KC_MINS,  KC_EQL,  KC_NO,    KC_BSPC,   \
       KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,   KC_T,   KC_Y,   KC_U,   KC_I,   KC_O,    KC_P,    KC_LBRC,  KC_RBRC,           KC_BSLS,   \
       KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,   KC_G,   KC_H,   KC_J,   KC_K,   KC_L,    KC_SCLN, KC_QUOT,  KC_NO,             KC_ENT,    \
-      KC_LSFT, KC_NO,   KC_Z,    KC_X,    KC_C,   KC_V,   KC_B,   KC_N,   KC_M,   KC_COMM, KC_DOT,  KC_SLSH,  KC_RSFT, KC_UP,    KC_UP,    \
+      KC_LSFT, KC_NO,   KC_Z,    KC_X,    KC_C,   KC_V,   KC_B,   KC_N,   KC_M,   KC_COMM, KC_DOT,  KC_SLSH,  KC_RSFT, KC_UP,    KC_SLSH,    \
       KC_LCTL, KC_LGUI, KC_LALT,                  KC_BSPC,KC_SPC, KC_DEL,                  KC_RGUI, F(0),    KC_LEFT, KC_DOWN,  KC_RIGHT),
 
   // 1: Function Layer
-  [1] = LAYOUT_all(
+  [1] = KEYMAP(
```
Comment (drashna) [inline keyboards/do60/keymaps/test/keymap.c]: Could these be changed back to `LAYOUT_all`?

**Comment by mechmerlin** [comment]
Hi Sennin, I sent you a message on Geekhack not too long ago about when I changed these over to LAYOUT to adhere to the new QMK standards. I also added new layouts to work with our new tool, the QMK Configurator: https://config.qmk.fm/#/do60/LAYOUT_chiwi60_default

**Comment by sennin32** [comment]
Hi,
Edited :)

**Comment by drashna** [comment]
Awesome! Thanks!

**Pull request merged**